### PR TITLE
lib: cleanup output.

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
- Don't use `/usr/bin/git` when `git` will do
- Output what method is being run each time

### CODE REVIEW

File: lib/step.rb [hunk 1]
```diff
@@ -21,6 +21,7 @@ def command_trimmed
              .join(" ")
              .gsub("#{HOMEBREW_LIBRARY}/Taps/", "")
              .gsub("#{HOMEBREW_PREFIX}/", "")
+             .gsub("/usr/bin/", "/")
```
Comment (Bo98) [inline lib/step.rb:24]: Wouldn't that replace `/usr/bin/git` with `/git`?

Comment (Bo98) [reply lib/step.rb:24]: How does this deal with something say in `/Library/Developer/CommandLineTools/usr/bin` (though maybe there's nothing that would do that though that needs worrying about)

Comment (MikeMcQuaid) [reply lib/step.rb]: Yes, yes it would 😁. Fixed!

Comment (MikeMcQuaid) [reply lib/step.rb]: > (though maybe there's nothing that would do that that needs worrying about)

there isn't but handled anyway now

**Pull request merged**